changelog/2025-05-12-mysql-connector-0-2-0.mdx

```diff
@@ -0,0 +1,32 @@
+---
+slug: '2025-05-12-mysql-connector-0-2-0'
+title: Conduit MySQL Connector v0.2.0
+draft: false
+tags: [conduit, connector, mysql, release]
+---
+
+[Conduit MySQL Connector v0.2.0](https://github.com/conduitio-labs/conduit-connector-mysql/releases/tag/v0.2.0) is now available, introducing new features, bug fixes, and improvements to enhance functionality and reliability.
+
+<!--truncate-->
+
+### Key Highlights
+
+
+- [**Record Batch Support and Multi-Collection Mode**](https://github.com/conduitio-labs/conduit-connector-mysql/pull/114). Added support for processing records in batches and introduced multi-collection mode for better flexibility by [@alarbada](https://github.com/alarbada).
+- [**New `noSnapshot` Source Configuration Parameter**](https://github.com/conduitio-labs/conduit-connector-mysql/pull/131). Introduced the `noSnapshot` parameter to allow users to skip the initial snapshot phase when configuring the source by [@alarbada](https://github.com/alarbada).
```

Comment:
This has been changed in https://github.com/conduitio-labs/conduit-connector-mysql/pull/141 (mentioned in the last bullet point), so I'd just mention the `snapshot.enabled` parameter.

Reply:
Actually, `noSnapshot` was already named `snapshot.enabled` in the PR mentioned in this bullet point, the PR description wasn't updated. Anyway, we can just mention that the parameter was added and what it does, possibly link both PRs.

Reply:
@lovromazgon good catch https://github.com/ConduitIO/conduit-site/pull/284/commits/2348134732b7c8277ae0fd3558f913fe3ea3b28f.